pyspectral/rsr_reader.py

```diff
@@ -334,25 +337,18 @@ def get_relative_spectral_responses(self, h5f):
                 self.set_band_central_wavelength_per_detector(h5f, bandname, dname)
 
 
-def check_and_download(**kwargs):
+def check_and_download(dest_dir=None, dry_run=False):
     """Do a check for the version and attempt downloading only if needed."""
-    dry_run = kwargs.get('dry_run', False)
-    dest_dir = kwargs.get('dest_dir', None)
-
     rsr = RSRDataBaseClass()
     if rsr.rsr_data_version_uptodate:
         LOG.info("RSR data already the latest!")
     else:
-        if dest_dir:
```

Comment:
Is this tested?

Reply:
None of the logic has changed and I didn't write any new tests. If you want me to add tests then I can look into doing that, but otherwise I think I would consider this a refactor (besides the aerosol_type deprecation).

Reply:
When doing refactoring it is of course preferable to have the coded covered by tests, so we are comfortable nothing breaks when refactoring. And it would be nice to have more test-coverage in general. But it would be a pity if no one dares touching the code because it is already poorly covered, so I just want you to give it a short consideration. I know I am to blame for the lack of test coverage here.

Reply:
I understand the concept, I just assumed things were tested already and didn't look to confirm. Given that this PR isn't urgent for me I can try to add it to my TODO list to try adding tests.